### DISCUSSION TITLE
data/2018/02/17/datascience-interivew-questions/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#80] [Comments]
#  데이터 사이언스 인터뷰 질문 모음집

https://zzsza.github.io/data/2018/02/17/datascience-interivew-questions/

**Comment by zzsza** [comment]
<p>빡시네</p>

>_This comment was made by Disqus. 2018-02-19 11:28:51 Keun Taek Lucas Han_

**Comment by zzsza** [comment]
<p>공부해야 될 게 엄청 많네요 ㅎㄷㄷ</p>

>_This comment was made by Disqus. 2018-03-01 03:46:34 whaledev_

**Comment by zzsza** [comment]
<p>안녕하세요. 항상 블로그 잘보고있습니다. 위 질문들을 개인적으로 사용해도 될지 문의드립니다. 감사합니다~!</p>

>_This comment was made by Disqus. 2018-11-06 06:05:38 JINSOL KIM_

**Comment by zzsza** [comment]
<p>감사합니당 :)</p>

>_This comment was made by Disqus. 2020-12-31 10:32:55 Yena jeon_

**Comment by zzsza** [comment]
<p>좋은 질문들이네요👍</p>

>_This comment was made by Disqus. 2021-08-22 07:01:34 asdf_

**Comment by unanchoi** [comment]
많이 도움되었습니다. 감사합니다.
